consensus/src/metrics.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+use prometheus::{default_registry, register_int_gauge_vec_with_registry, IntGaugeVec, Registry};
+
+#[derive(Clone, Debug)]
+pub struct ConsensusMetrics {
+    /// The number of elements in the Dag (for Tusk or Bullshark)
+    pub consensus_dag_size: IntGaugeVec,
+    /// The last committed round from consensus
+    pub last_committed_round: IntGaugeVec,
+    /// The number of elements from the vertices secondary index (external consensus)
+    pub external_consensus_dag_vertices_elements: IntGaugeVec,
```

Comment:
Because we always derive the size of the vertices map and we report them to metrics - if we would use the `IntCounterVec` we would end up always adding up the sizes one on top of the other. Now we just always report the total which `gauge` is for

Reply:
Same stands for the other metrics